content/en/blog/2023/otel-go-metrics-sdk-stable.md

```diff
@@ -0,0 +1,18 @@
+---
+title: First stable release of OTel Go metric SDK
+linkTitle: Go Metric SDK is Stable
+date: 2023-09-29
+author: '[Fabrizio Ferri-Benedetti](https://github.com/theletterf/) (Splunk)'
+cSpell:ignore: Benedetti Fabrizio Ferri
+---
+
+The Go metric SDK has finally reached stable status with version 1.19.0 of
+OpenTelemetry Go. Project stability guarantees now apply to the
+[go.opentelemetry.io/otel/sdk/metric](https://pkg.go.dev/go.opentelemetry.io/otel/sdk/metric)
+package.
+
+To get started with OTel Go metrics,
+[read the documentation](/docs/instrumentation/go/manual) or browse the
+[examples](https://github.com/open-telemetry/opentelemetry-go/tree/main/example)
+for demonstrations of different instrumentation scenarios covered by the
+OpenTelemetry Go Instrumentation.
```

Comment:
I think we can simply refer to these instead
- `/docs/instrumentation/go/getting-started`
- `/docs/instrumentation/go/manual#metrics`

E.g.
```suggestion
To learn more check the [getting started](/docs/instrumentation/go/getting-started)
and [manual metrics](/docs/instrumentation/go/manual#metrics) documentation.
```

The getting started has reference to examples.